python-threatexchange/threatexchange/exchanges/clients/fb_threatexchange/api.py

```diff
@@ -16,7 +16,7 @@
 
 import requests
 from requests.packages.urllib3.util.retry import Retry
-from threatexchange.exchanges.clients.utils.common import TimeoutHTTPAdapter
+from ..utils.common import TimeoutHTTPAdapter
```

Comment:
blocking: I've had too many painful experiences with relative imports, and now always recommend absolute imports.